spec/bitrate.spec.js

```diff
@@ -0,0 +1,30 @@
+import * as Bitrate from '../src/parser/bitrate';
+
+describe('updateEstimatedBitrate', () => {
+  beforeEach(() => {
+    Bitrate.estimatedBitrate = 0;
+  });
+
+  it('doesn\'t update estimated bitrate if one value is missing', () => {
```

Comment:
Usually I don't like to switch from simple to double quotes and prefer to stick with the escape because it brings inconsistency. I'll replace with "does not" :trollface: